go.mod

```diff
@@ -105,7 +105,7 @@ require (
 
 replace (
 	git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
-	github.com/briandowns/spinner => github.com/alonyb/spinner v1.12.1
+	github.com/briandowns/spinner => github.com/alonyb/spinner v1.12.6
```

Comment:
You are missing the matching update in go.sum

```diff
-github.com/alonyb/spinner v1.12.1/go.mod h1:QOuQk7x+EaDASo80FEXwlwiA+j/PPIcX3FScO+3/ZPQ=
+github.com/alonyb/spinner v1.12.6/go.mod h1:mQak9GHqbspjC/5iUx3qMlIho8xBS/ppAL/hX5SmPJU=
```

Reply:
thanks @afbjorklund it's done :)